assets/scss/modules/_buttons.scss

```diff
@@ -224,19 +224,32 @@
       position: relative;
       right: 3px;
       top: 0px;
+      padding: 14px 30px 13px 22px;
 
       &:hover,
       &:active {
         background: $dp-color-white;
       }
 
       &:focus {
-        box-shadow: inset 0 0 0 2px $dp-color-orange-focus,
+        box-shadow: inset 0 0 0 1px $dp-color-orange-focus,
           inset 0 0 0 3px $dp-color-white;
       }
       &.active {
         background: $dp-color-white;
       }
+
+      .dp-dropdown-arrow {
+        border-style: solid;
+        height: 0;
+        width: 0;
+        border-width: 6px 5.5px 0 5.5px;
```

Comment:
Dejo un artículo acerca de fractional pixels y sub-pixel rendering de los diferentes browsers.
[https://cruft.io/posts/percentage-calculations-in-ie/](https://cruft.io/posts/percentage-calculations-in-ie/)
Particularmente no suelo usar decimales en pixels a menos que venga de un resultado de algún cálculo de JS.